associations/tests/page/base_test_page.py

```diff
@@ -0,0 +1,54 @@
+from associations.tests.base_test import BaseTestCase
+
+# Please see the comments on test_page.yaml to get a better understanding of the test fixtures.
+
+ALL_USERS = ['17admin_biero', '17member_biero', '17page_biero', '17admin_pdm', '17member_pdm', '17page_pdm']
+"""A list of user ids covering all the spectrum of roles and permissions."""
+
+ALL_USERS_EXCEPT_PAGE_BIERO = [user for user in ALL_USERS if user != '17page_biero']
+"""Same as ALL_USERS, but with 17page_biero removed."""
+
+ALL_USERS_EXCEPT_PAGE_ADMIN = [user for user in ALL_USERS if 'page' not in user]
+"""Same as ALL_USERS, but with 17page* removed."""
+
+ALL_USERS_EXCEPT_ADMIN = [user for user in ALL_USERS if 'admin' not in user]
+"""Same as ALL_USERS, but with all the association administrators removed."""
+
+
+class BasePageTestCase(BaseTestCase):
```

Comment:
Many redundancies with the News tests, maybe there is a way to refactor it

Reply:
Yeah. As we previously discussed, a way to do it would be to merge `News` and `Page` into a `Page` model, only adding a `type` field which would store the… type of page.

Reply:
This will be done… one day. For now, let's keep the tests as they are ~